Alicerce/Sources/Logging/Providers/FileProvider.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  FileProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 03/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+final class FileProvider {
+    
+    internal var minLevel: Log.Level = .error
+    internal var formatter: LogItemFormatter = LogItemStringFormatter()
+    
+    let fileURL: URL
+    let fileManager = FileManager.default
+    
+    //MARK:- Lifecycle
+    
+    init(fileURL: URL) {
+        self.fileURL = fileURL
+    }
+    
+    //MARK:- Public Methods 
+    
+    public func clear() {
```

Comment:
I see your point, but then I think we should at least check if we can write to the file on `init` or something. Otherwise the user will only notice that the logs aren't working by looking at the console, which I think isn't a very good experience.